docs/reference/sql/greptime-private/overview.md

```diff
@@ -0,0 +1,14 @@
+---
+keywords: [system tables, greptime private, pipelines, slow queries]
+description: The overview of system tables in the `greptime_private` database.
+---
+
+# Greptime Private
+
+GreptimeDB stores some important internal information as system tables in the `greptime_private` database. These system tables are controlled by GreptimeDB, and the users can only query them but cannot modify them.
```

Comment:
```suggestion
GreptimeDB stores some important internal information as system tables in the `greptime_private` database. These system tables are controlled by GreptimeDB, and you can only query them but cannot modify them.
```

Reply:
It's not accurate, since `pipelines` is also in `greptime_private` and can be modified.